.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - "2.7"
+
+env:
+  - SIMPHONY_VERSION=0.2.0
+  - SIMPHONY_VERSION=master
+matrix:
+  allow_failures:
+    - env: SIMPHONY_VERSION=master
+
+install:
+  - pip install -r dev_requirements.txt
+  - pip install git+https://github.com/simphony/simphony-common.git@${SIMPHONY_VERSION}#egg=simphony
+  - python setup.py develop
+
+script:
+  # - flake8 
```

Comment:
For SimPhoNy, we are ensuring that code styling is checked by using flake8.  The line should be changed then from `# - flake8` to `- flake8 .`  

I did not enable the flake8 test when originally creating the `.travis.yml`as there are many minor things that have to be changed to the code.